tests/test_cameras.py

```diff
@@ -117,6 +118,18 @@ def setUp(self) -> None:
         torch.manual_seed(42)
         np.random.seed(42)
 
+    def test_look_at_view_transform_from_eye_point_tuple(self):
+        dist = math.sqrt(2)
```

Comment:
Add a comment here that you are checking the case where `eye` as well as dist, elev, azim are provided. 

Add another test case where only `eye` is provided e.g. 
```
R_eye_only, t_eye_only = look_at_view_transform(eye=eye)
```

Reply:
hi @nikhilaravi , there will need to be default values for dist, elev azim to pass only eye. I think this is a better interface, and will not break existing code that uses those values. I'll make those changes and let you review.